great_expectations/validator/validation_statistics.py

```diff
@@ -0,0 +1,39 @@
+from typing import NamedTuple
+
+from great_expectations.core.expectation_validation_result import (
+    ExpectationValidationResult,
+)
+
+
+class ValidationStatistics(NamedTuple):
+    evaluated_expectations: int
+    successful_expectations: int
+    unsuccessful_expectations: int
+    success_percent: float | None
+    success: bool
+
+
+def calc_validation_statistics(
+    validation_results: list[ExpectationValidationResult],
+) -> ValidationStatistics:
+    """
+    Calculate summary statistics for the validation results and
+    return ``ExpectationStatistics``.
+    """
```

Comment:
Do we envision this ever living on an object?

Reply:
IDK I thought about just making it a static method on `ValidationStatistics`. I don't really have strong opinions though. Seems like wrapping it in another class might be YAGNI though.